src/main/paradox/docs/release-notes/index.md

```diff
@@ -0,0 +1,230 @@
+
+Introducing these boundaries has opened the door to make several performance and functional improvements to the system
+as follows:
+
+*   data indexing can now be configured at the project level without impacting the overall system; we've introduced a
+    new resource type _View_ that controls how and where the data is being transformed. They can be managed at runtime
+    by the clients with "administrative" privileges enabling the development of applications specific to the area of
+    interest. An example of that is the [BBP Nexus Search] application designed specifically to address the needs of
+    the BlueBrain Project.
+*   indexing processes are now created for each individual project increasing the indexing throughput and allowing
+    the use of distinct indexing targets.
+*   the uniqueness of a resource within the system was determined by the resource id which comprised of the
+    "organization", "domain", "schema" name and version and the resource id. Introducing these data boundaries allowed
+    us to relax this constraint and allow multiple resources to share the same id if they are scoped in different
+    projects.
+*   access control lists are now restricted to either root (`/`), organization (`/{org}`) or project
+    (`/{org}/{project}`) removing the need to index these definitions along with the data.
+    
+#### Client defined identifiers
+
+The `v0.m.p` series was very opinionated on how resource ids are defined and prohibited client provided identifiers. The
+choice at the time was that a resource id needs to be resolvable (the resource id needed to match the url to access it).
+This strong constraint tied the resource ids to the Nexus deployment where they were managed. While this was probably
+fine for most data within the system, it was nearly impossible to manage data from external sources that came with its
+own identification scheme (e.g. ontologies).
+
+The decision was made to remove this constraint and allow clients to specify their own identifiers. In order to
+maintain the same API simplicity with respect to accessing resources we came up with a aliasing and curie scheme
+configurable at the project level (_apiMappings_) that handles bidirectional compaction and expansion of resource
+identifiers.
+
+A simple http proxying configuration with URL rewriting deployed in front of the Nexus API allows resources to continue
+to be resolvable.
+
+#### Resolution mechanisms for shared resources
+
+During the past year of production use of Nexus we have noticed that users tend to develop schemas and contexts as
+reusable components. An example of that is the [Neuroshapes] initiative, a community effort for a shared vocabulary and
+collection of constraints for neuroscience, an initiative under the [INCF] umbrella.
+
+The use of schemas and contexts in the `v0.m.p` series applied a restriction on the locality of constrained resources,
```

Comment:
Worth mentioning here that contexts are no longer first class citizen resources in v1. Also make clear that they can still be created as data.